Source/Core/DolphinNoGUI/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+if(NOT(USE_X11 OR ENABLE_HEADLESS))
```

Comment:
It's just a weird condition, and there's confusion about the nogui / headless situation. I wonder how to clarify that.